libs/partners/nvidia-trt/langchain_nvidia_trt/llms.py

```diff
@@ -218,7 +218,7 @@ def _invoke_triton(self, model_name, inputs, outputs, stop_words):
         result_queue = StreamingResponseGenerator(
             self,
             request_id,
-            force_batch=False,
+            signal_stop=False,
```

Comment:
renaming to meaningful. Inverting it at line 405. 
TLDR triton doesn't let one cancel a request https://github.com/triton-inference-server/server/issues/4818